### DISCUSSION TITLE
Any way to create dynamic collections (or regular collections, even) as sorted by top folder location within a library?

### DISCUSSION THREAD

**Discussion by KingArthurHS** [meisnate12/Plex-Meta-Manager-Configs#177] [Q&A]
I have a question regarding a use-case that this utility is certainly not designed for, but may be able to help with.

I have a Plex library that is a mish-mash of assorted video and audio files, mostly from old game, audio, and video projects. This library has like 5,000 pieces of media in it, organized in dated folders. I use Plex to navigate around this library when I'm looking for something so I'm not opening instances of VLC or MPC over and over.

In Plex, I always have to use Folder view because any kind of ordering Plex will do to try and match some Movie or TV DB would obviously fail spectacularly. Folder view SUCKS though, because there's no way to view in a tile-view with generated thumbnails, which would dramatically speed up my hunting for some video clip that I need.

If I could use PMM to generate collections where the defining criteria is just the top folder name where a given file is located, that would be great. However, I can't seem to find an attribute that will let me do this. Any thought? A dynamic collection would be best, but it's probably worth it to me to slog through making like 400 manual collections if this is doable.

Also, apologies if some of my terminology is incorrect or not descriptive. I'm trying!

Thanks!

**Comment by meisnate12** [comment]
you can probably do this with a `plex_all: true` and using the filepath filter

**Reply by KingArthurHS** [reply]
> you can probably do this with a `plex_all: true` and using the filepath filter

Ahhhhhh very good. I am now like 99% functional.

What would my syntax need to be in order to include the contents of multiple folders within a single collection?

For example, I've tried

collections:
    FolderName:
        sync_mode: sync
        plex_all: true
        filters:
            filepath: FolderName

and it's been successful to include that one folder as a collection. But if I want to include two folder within that collection, how should I format that?

collections
    FolderGroup:
        sync_mode: sync
        plex_all: true
        filters:
            filepath: FolderName1, FolderName 2

Or do I need a separate filepath entry line for each folder name? I'd also tried
filepath: ("FolderName1", "FolderName2")

Neither has worked to achieve this. Thanks for your help!
